javascript.js

```diff
@@ -137,4 +137,23 @@ form.addEventListener('submit', (event) => {
   } else {
     form.submit();
   }
-});
\ No newline at end of file
+});
+
+const email = document.querySelector('.email');
+email.addEventListener('input', () => {
+  const emailValue = email.value;
+  localStorage.setItem('emailValue', emailValue);
+});
+
+if (localStorage.getItem('emailValue')) {
+  email.value = localStorage.getItem('emailValue');
+}
+const myName = document.querySelector('.name');
+myName.addEventListener('input', () => {
+  const myNameValue = myName.value;
+  localStorage.setItem('myNameValue', myNameValue);
+});
+
+if (localStorage.getItem('myNameValue')) {
+  myName.value = localStorage.getItem('myNameValue');
+}
```

Comment:
- [ ] Hello @team, Kudos working on the localstorage part of the project, currently i can only save the name and the email data in the `Local Storage` while the `body (i.e the text-area)` is not store in the `Local Storage` and this doesn't go in-line with what is required in the project requirement.

Kindly fix it and be sure the `body (i.e the text-area)` value is also stored in the `Local Storage`